docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx

```diff
@@ -0,0 +1,290 @@
+---
+title: Platform
+sidebar_label: Platform
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+Every robot platform has unique structures such as versatile sensor mounting solutions, wireless charging receivers, and waterproofing enclosures; we refer to these as attachments.
+
+## Joystick Controller
+We support two types of controllers:
+  - **ps4**: standard Playstation4 controller.
+  - **logitech**: Logitech F710
+
+```yaml
+controller: ps4 # or logitech
+```
+
+## Attachments
+Each platform has its own unique set of attachments. To facilitate customizability, any attachment from any platform can be added on any other platform.
```

Comment:
- Change to _customization_. 
- Change _platform_ to _robot platform_.

Reply:
Done